src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -637,6 +640,11 @@ Parameter Engine::GetMetric(const std::string& name, const std::map<std::string,
     } else if (name == ov::caching_properties) {
         std::vector<ov::PropertyName> cachingProperties = { ov::device::full_name };
         return decltype(ov::caching_properties)::value_type(cachingProperties);
+    } else if (name == ov::intel_cpu::denormals_optimization) {
+        return decltype(ov::intel_cpu::denormals_optimization)::value_type(engConfig.denormalsOptMode == Config::DenormalsOptMode::DO_On);
+        return engConfig.denormalsOptMode;
+    } else if (name == ov::intel_cpu::sparse_weights_decompression_rate) {
+        return decltype(ov::intel_cpu::sparse_weights_decompression_rate)::value_type(engConfig.fcSparseWeiDecompressionRate);
```

Comment:
do we have tests for it?

Reply:
@antonvor Take this PR as reference for tests: https://github.com/openvinotoolkit/openvino/pull/16106. The only difference the test shouldn't be shared, but CPU specific.